pkg/eval/json_evaluator_model.go

```diff
@@ -8,6 +8,17 @@ type Flags struct {
 	Flags map[string]Flag `json:"flags"`
 }
 
+func (f Flags) Merge(ff Flags) Flags {
```

Comment:
might be worth mentioning this behaviour in the README / -f flag definition.
I believe the only reference to this being first come first serve is in the original issue.

Reply:
Agreed, good catch @James-Milligan . First come, first server makes the sense to me but that behavior should be documented. Thanks for the PR @moshebe!

Reply:
Additionally, could you add an example of starting flagd with multiple filepath syncs?

Reply:
Thanks for the feedback @James-Milligan, @beeme1mr and @toddbaert.
Added an example to the README and describing the option within the start command (`--help`)

Please review the texts 🙏